contracts/generic/MultiSig.sol

```diff
@@ -84,16 +84,24 @@ contract MultiSig {
         Script storage script = scripts[scriptAddress];
         require(script.state == ScriptState.Approved, "script state must be Approved");
 
-        // passing scriptAddress to allow called script access its own public fx-s if needed
-        if (scriptAddress.delegatecall.gas(gasleft() - 23000)
-            (abi.encodeWithSignature("execute(address)", scriptAddress))) {
-            script.state = ScriptState.Done;
-            result = true;
-        } else {
-            script.state = ScriptState.Failed;
-            result = false;
+        result = _execute(scriptAddress);
+    }
+
+    function dryExecute(address scriptAddress) public returns (bool result) {
```

Comment:
If it always reverts (never actually returns the bool result), maybe it would be more clear to declare it as "void".